### PR TITLE
sqsgenerator

### PR DESCRIPTION
**Pull request by jan-janssen**
@conda-forge/help-python-c

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
Waiting for https://github.com/conda-forge/staged-recipes/pull/8888 to be merged

File: recipes/sqsgenerator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{% set name = "sqsgenerator" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dnoeger/sqsgenerator/archive/master.zip
```
Comment (isuruf) [inline recipes/sqsgenerator/meta.yaml]: Don't use master. Use a stable release

Comment (scopatz) [reply recipes/sqsgenerator/meta.yaml]: It doesn't have to be stable, just a release that isn't an rc or pre-release

File: recipes/sqsgenerator/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  sha256: d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f
+
+build:
+  number: 1
```
Comment (scopatz) [inline recipes/sqsgenerator/meta.yaml]: The build number should be zero

**Comment by jan-janssen** [comment]
@dnoeger Is it ok if I add you as a maintainer.

**Comment by dnoeger** [comment]
@jan-janssen yes of course, many thanks

File: recipes/sqsgenerator/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  host:
+    - python
+    - pip
+    - gmp 
+    - cython
+    - pymatgen
+  run:
+    - python
+    - gmp 
+    - cython
```
Comment (scopatz) [inline recipes/sqsgenerator/meta.yaml]: does this really need cython at run time?

File: recipes/sqsgenerator/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    - pymatgen
+
+test:
+  imports:
+    - sqsgenerator
+
+about:
+  home: https://github.com/dnoeger/sqsgenerator
+  license: OTHER
```
Comment (scopatz) [inline recipes/sqsgenerator/meta.yaml]: what is the name of the license?

File: recipes/sqsgenerator/meta.yaml (hunk 3 above)
Comment (jan-janssen) [reply recipes/sqsgenerator/meta.yaml]: You are right - it is not required at runtime - so I moved it to the `build` section.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**